server/sonar-web/src/main/js/apps/webhooks/components/DeliveryItem.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import CodeSnippet from '../../../components/common/CodeSnippet';
+import DeferredSpinner from '../../../components/common/DeferredSpinner';
+import { formatMeasure } from '../../../helpers/measures';
+import { translateWithParameters, translate } from '../../../helpers/l10n';
+import { WebhookDelivery } from '../../../app/types';
+
+interface Props {
+  className?: string;
+  delivery: WebhookDelivery;
+  loading: boolean;
+  payload: string | undefined;
+}
+
+export default function DeliveryItem({ className, delivery, loading, payload }: Props) {
+  return (
+    <div className={className}>
+      <p className="spacer-bottom">
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2Fadministration-console-for-webhooks&issues=AWHDO7-P1hZr15o4gqrG&open=AWHDO7-P1hZr15o4gqrG)

[ISSUE_KEY]: AWHDO7-P1hZr15o4gqrG